irmaseal-cli/src/cli.yml

```diff
@@ -35,10 +35,6 @@ subcommands:
         help: Sets the input file to use
         required: true
         index: 1
-    - OUTPUT:
```

Comment:
Is it really practical to remove this? If I have a file that is named exactly the same as a file within the tar, what happens then? Is it overwritten then or do I get an error?